pom.xml

```diff
@@ -175,7 +175,7 @@
             <dependency>
                 <groupId>io.github.microutils</groupId>
                 <artifactId>kotlin-logging-jvm</artifactId>
```

Comment:
We actually haven't planned it (yet), because slf4j has been enough. I believe we can unify logging in all modules, remove explicit slf4j-api dependency and use kotlin-logging instead